docs/interpretation_of_JSON_Schema_draft_7.md

```diff
@@ -54,7 +54,10 @@ If only one side has a property defined, it is used as is, if both have it defin
 - `additionalProperties` if both models contain it the two are recursively merged together. 
 - `patternProperties` if both models contain a pattern the corresponding models are recursively merged together. 
 - `properties` if both models contain the same property the corresponding models are recursively merged together. 
-- `items` if both models contain items they are recursively merged together.
+- `items` are merged together based on a couple of rules:
+    - If both models are simple arrays those item models are merged together as is.
+    - If both models are tuple arrays each tuple model (at specific index) is merged together.
+    - If either one side is different from the other, the simple array is merged together with each tuple model.
```

Comment:
Do you think it's a correct behaviour? If we found tuple and we have defined items as standalone schema, why do you want merge models? I think we should override only when two models are tuple or single schema, otherwise, tuple should override items.

Reply:
You are right... Also, tuples should ONLY be considered when `additionalItems` is false (Default is `true`), otherwise, you are allowed to have multiple values in your array so it can no longer be considered a tuple.

Reply:
Unless `maxItems` is the same length as the `items` defined, then it can be considered a tuple.

Reply:
Ohh the posibilities 😅

Reply:
> You are right... Also, tuples should ONLY be considered when additionalItems is false (Default is true), otherwise, you are allowed to have multiple values in your array so it can no longer be considered a tuple.

I think that if you have `additionalItems` set to true, then `tuple` are handled as prefix items :)

Reply:
@magicmatatjahu can you clarify what you mean by prefix items?

Reply:
when you have (eg) `items: [{ type: string }, { type: number }]` and `additionalItems : true` then you can define array as ["foo", 1, "something"], so `items` are treated as "prefix" items :)

Reply:
Ahh yes, added another issue for further discussion https://github.com/asyncapi/generator-model-sdk/issues/224